lib/features/composer/presentation/composer_controller.dart

```diff
@@ -401,6 +399,35 @@ class ComposerController extends BaseController with DragDropFileMixin {
     });
   }
 
+  Future<CreateEmailRequest> _generateCreateEmailRequest() async {
```

Comment:
we should bring down the `pre-check` condition too, because we use a lot `!` in this function

Reply:
Done